lib/assets/javascripts/twine.coffee

```diff
@@ -124,7 +125,9 @@
     elements[node.bindingId] ?= {}
 
   # Queues a refresh of the DOM, batching up calls for the current synchronous block.
-  Twine.refresh = ->
+  # The callback will be called once when the refresh has completed.
+  Twine.refresh = (callback) ->
+    refreshCallbacks.push(callback) if callback
     return if refreshQueued
     refreshQueued = true
     setTimeout(Twine.refreshImmediately, 0)
```

Comment:
Just wondering, could we do `Twine.refreshImmediately.bind(Twine, callback)` here and `Twine.refreshImmediately = (callback) ->` in https://github.com/Shopify/twine/pull/91/files#diff-5c7054f0eed5458018a14692e6e48d93R139?

That way we wouldn't need the `refreshCallbacks` variable and both `refresh` and `refreshImmediately` would have the same api interface